README.md

```diff
@@ -1 +1,253 @@
+  </ul>
+</details>
+
+<details>
+<summary>Github</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+<details>
+ <summary>Linter instullation usage</summary>
+ </details>
+ <details>
+ <summary>Flex and it's management</summary>
+ </details>
+<p></p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps:
+
+### Prerequisites
+
+In order to run this project you need:
+
+<details>
+ <summary>VS Code</summary>
+ </details>
+ <details>
+ <summary>Github</summary>
+ </details>
+ <details>
+ <summary>Git</summary>
+ </details>
+
+### Setup
+
+Clone this repository to your desired folder: Using git clone
+
+
+### Install
+
+Install this project with: cd Mobile-First-Portfolio
+
+<!--
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+--->
+
+### Usage
+
+To run the project, execute the following command: npm command
+>GitHub Pages Server
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
+
+### Run tests
+
+To run tests:  run the following command:
+>npx stylelint "**/*.{css,scss}"
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+### Deployment
+
+You can deploy this project using git command:
+>git@github.com:Ejikeokegbe/Mobile-First-Portfolio.git
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+<h2>Okegbe Clifford Ejike</h2>
+
+- GitHub: [@gEjikeokegbe](https://github.com/Ejikeokegbe)
+
+- LinkedIn: [Ejikeokegbe](https://linkedin.com/in/Ejikeokegbe)
+
+
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **Making it user friendly**
+- [ ] **Responsive**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Readers Your support to the project on this channel:
+>git@https://github.com/Ejikeokegbe/Hello-Microverse-Project.git
+
+If you like this project support through the link.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+I would like to thank Microverse for their good job
+
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+
+- **[Question_1]**
+
+  - [Answer_1]
+
+- **[Question_2]**
+
+  - [Answer_2]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [optional] please, your MIT link is broken. Kindly set it up .we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish